README.md

```diff
@@ -63,6 +64,9 @@ A list of links of Academic references and research papers related to Blockchain
 - https://nakamotoinstitute.org/the-idea-of-smart-contracts/
 - https://nakamotoinstitute.org/formalizing-securing-relationships/
 - https://nakamotoinstitute.org/contract-language/
+ 
+## Smart Contracts Security
+- [Demystifying Exploitable Bugs in Smart Contracts 2021-2022](https://www.cs.purdue.edu/homes/zhan3299/res/ICSE23.pdf)
```

Comment:
Can you do the same format as the other please:

Author name (Year), Title

So it looks more like Harvard or APA referencing style.

(The idea is that scholars can copy paste easily in the reference list 🙂)